portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase

```diff
@@ -1092,97 +1280,120 @@ definition {
 		property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
 		property testray.component.names = "Training";
 
-		Navigator.openURL();
+		task ("Given: User adds a new private page") {
+			Navigator.openURL();
 
-		ProductMenu.gotoPortlet(
-			category = "Site Builder",
-			portlet = "Pages");
+			ProductMenu.gotoPortlet(
+				category = "Site Builder",
+				portlet = "Pages");
 
-		PagesAdmin.addPrivatePage(pageName = "Private Page");
+			PagesAdmin.addPrivatePage(pageName = "Private Page");
+		}
 
-		Navigator.gotoPrivateSitePage(
-			pageName = "Private Page",
-			siteName = "Guest");
+		task ("When: User export the private page") {
+			Navigator.gotoPrivateSitePage(
+				pageName = "Private Page",
+				siteName = "Guest");
 
-		LAR.exportSiteCP(
-			larFileName = "ExportPrivatePage",
-			privateLayout = "true",
-			siteScopeName = "Guest");
+			LAR.exportSiteCP(
+				larFileName = "ExportPrivatePage",
+				privateLayout = "true",
+				siteScopeName = "Guest");
 
-		var key_larFileName = "${larFileName}";
-		var larFileName = LAR.getLarFileName();
+			var key_larFileName = "${larFileName}";
+			var larFileName = LAR.getLarFileName();
 
-		LAR.downloadLar();
+			LAR.downloadLar();
+		}
 
-		JSONGroup.addGroup(groupName = "Site Name");
+		task ("And: Add a new site") {
```

Comment:
Put this action into previous task, no need to split them.
Example:
"When: User export the private page and add a new site"